index.html

```diff
@@ -4778,41 +4789,28 @@ <h2 id="AudioBufferSourceNode">
           </dt>
           <dd>
             <p>
-              An optional value in seconds where looping should begin if the
-              <code>loop</code> attribute is true. Its default
+              An optional <a>playhead position</a> where looping should begin
```

Comment:
It allowed me to remove several instances of language having to do with multiplying by the sample rate, which you didn't like -- see https://github.com/WebAudio/web-audio-api/pull/1143/commits/73a9fd679106fc47ecba44d37e05ac64be2c505e at line 4802